production_services/settings.py

```diff
@@ -100,7 +101,7 @@
         'ENGINE': 'django.db.backends.mysql',
         'NAME': os.getenv('USER_DB_DATABASE', 'ensembl_production_services'),
         'USER': os.getenv('USER_DB_USER', 'ensembl'),
-        'PASSWORD': os.getenv('USER_DB_PASSWORD', ''),
+        'PASSWORD': os.getenv('USER_DB_PASSWORD', 'ensembl'),
```

Comment:
Is it what it looks like ? You know `ensembl-production-services` is a public repository, right ?

Reply:
Yes I know.